.github/workflows/release-artifactregistry.yml

```diff
@@ -90,6 +90,15 @@ jobs:
         with:
           node-version: 18
 
+      - uses: goreleaser/goreleaser-action@v5
```

Comment:
there seems to already be v6 for this action.
probably better to bump in a different PR later on.